yt/frontends/ramses/field_handlers.py

```diff
@@ -23,14 +23,18 @@ def register_field_handler(ph):
 DETECTED_FIELDS = {}
 
 
-class RegisteredRAMSESFieldFileHandler(abc.ABCMeta):
+class RegisteredRAMSESFieldFileHandler:
     """
     This is a base class that on instantiation registers the file
     handler into the list. Used as a metaclass.
     """
 
-    def __new__(meta, name, bases, class_dict):
-        cls = abc.ABCMeta.__new__(meta, name, bases, class_dict)
+    def __init_subclass__(cls, *args, **kwargs):
```

Comment:
I think this is not how this pattern is supposed to be used. From @matthewturk 's PR I think the idea was to remove the RegisteredStuff meta classes and put this special method in subclasses

Reply:
My bad!

Reply:
Should be fixed now